client/state/selectors/get-reader-follows.js

```diff
@@ -15,7 +15,7 @@ import createSelector from 'lib/create-selector';
  * @return {Integer} Follow count
  */
 const getReaderFollows = createSelector(
-	state => values( state.reader.follows.items ),
+	state => values( filter( state.reader.follows.items, item => ! item.error ) ),
```

Comment:
> will this end up removing the error from the subscription list on following/manage?

Yep - that's intended behaviour I think? Unless we later need the error information in the UI.